src/Gui/Stylesheets/Dark-blue.qss

```diff
@@ -1,8 +1,8 @@
 /*
 ABOUT
 ============================================================================================================
-version 2.20
-Qt theme (stylesheet) specially developed for FreeCAD (https://www.freecadweb.org/).
+version 2.21
+Qt theme (stylesheet) specially developed for FreeCAD (https://www.freecadweb.org//).

```

Comment:
you've added an extra `/` to the end of the URL :face_with_head_bandage: :laughing:  
You did this also for the others